lib/KaApi.php

```diff
@@ -0,0 +1,142 @@
+<?php
+
+// Copyright 2022. Plesk International GmbH. All rights reserved.
+
+namespace WHMCS\Module\Server\Plesk360Monitoring;
+
+use GuzzleHttp\Client;
+
+class KaApi
+{
+    private const CONTENT_TYPE_JSON = 'application/json';
+
+    private $client;
+
+    public function __construct(array $params)
```

Comment:
just explode params, no need in the $params black box

Reply:
Done